projects/scion/toolkit/uuid/src/uuid.util.ts

```diff
@@ -9,26 +9,36 @@
  */
 
 /**
- * Allows generating pseudo-random identifiers.
+ * Fast UUID generator, RFC4122 version 4 compliant.
```

Comment:
Please move this description to the namespace:

```ts
/**
 * Fast UUID generator, RFC 4122 version 4 compliant.
 *
 * Usage:
 *
 * ```ts
 * UUID.randomUUID();
 * ```
 */
export namespace UUID {
}
```

Reply:
There is no `namespace` anymore.

As already commented: I added re-export to `public_api.ts` as `export * as UUID from './uuid.util';` and eliminated `namespace`. Namespaces are outdated constructs and not recommended from TypeScript itself. They generate IIFE (immediately invoked function expression) which are executed at load time and not tree-shakeable as well.

Everything works fine. You can use the functionality in the same manner as `UUID.randomUUID()`.
If you still want to have namespaces, I can revert re-exporting and add `namespace` again.